src/databases/Blueprint/avtBlueprintWriter.h

```diff
@@ -0,0 +1,109 @@
+* full copyright notice is contained in the file COPYRIGHT located at the root
+* of the VisIt distribution or at http://www.llnl.gov/visit/copyright.html.
+*
+* Redistribution  and  use  in  source  and  binary  forms,  with  or  without
+* modification, are permitted provided that the following conditions are met:
+*
+*  - Redistributions of  source code must  retain the above  copyright notice,
+*    this list of conditions and the disclaimer below.
+*  - Redistributions in binary form must reproduce the above copyright notice,
+*    this  list of  conditions  and  the  disclaimer (as noted below)  in  the
+*    documentation and/or other materials provided with the distribution.
+*  - Neither the name of  the LLNS/LLNL nor the names of  its contributors may
+*    be used to endorse or promote products derived from this software without
+*    specific prior written permission.
+*
+* THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT  HOLDERS AND CONTRIBUTORS "AS IS"
+* AND ANY EXPRESS OR  IMPLIED WARRANTIES, INCLUDING,  BUT NOT  LIMITED TO, THE
+* IMPLIED WARRANTIES OF MERCHANTABILITY AND  FITNESS FOR A PARTICULAR  PURPOSE
+* ARE  DISCLAIMED. IN  NO EVENT  SHALL LAWRENCE  LIVERMORE NATIONAL  SECURITY,
+* LLC, THE  U.S.  DEPARTMENT OF  ENERGY  OR  CONTRIBUTORS BE  LIABLE  FOR  ANY
+* DIRECT,  INDIRECT,   INCIDENTAL,   SPECIAL,   EXEMPLARY,  OR   CONSEQUENTIAL
+* DAMAGES (INCLUDING, BUT NOT  LIMITED TO, PROCUREMENT OF  SUBSTITUTE GOODS OR
+* SERVICES; LOSS OF  USE, DATA, OR PROFITS; OR  BUSINESS INTERRUPTION) HOWEVER
+* CAUSED  AND  ON  ANY  THEORY  OF  LIABILITY,  WHETHER  IN  CONTRACT,  STRICT
+* LIABILITY, OR TORT  (INCLUDING NEGLIGENCE OR OTHERWISE)  ARISING IN ANY  WAY
+* OUT OF THE  USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH
+* DAMAGE.
+*
+*****************************************************************************/
+
+// ************************************************************************* //
+//                              avtBlueprintWriter.h                               //
+// ************************************************************************* //
+
+#ifndef AVT_BLUEPRINT_WRITER_H
+#define AVT_BLUEPRINT_WRITER_H
+
+#include <avtDatabaseWriter.h>
+
+#include <string>
+#include <vector>
+
+#include "conduit.hpp"
+
+class DBOptionsAttributes;
+
+// ****************************************************************************
+//  Class: avtBlueprintWriter
+//
+//  Purpose:
+//      A module that writes out blueprint files.
+//
+//  Programmer:
+//  Creation:
+//
+//  Modifications:
+//
+//
+// ****************************************************************************
+
+class
+avtBlueprintWriter : public virtual avtDatabaseWriter
+{
+  public:
+                   avtBlueprintWriter();
+    virtual       ~avtBlueprintWriter() {;};
+
+  protected:
+    static int            INVALID_CYCLE;
+    static double         INVALID_TIME;
+
+    std::string    stem;
```

Comment:
m_ prefix would be good (especially for things like "time")

Reply:
done